docs/QbeastFormat.md

```diff
@@ -72,7 +72,7 @@ Let's take a look to the Index metadata to understand what is written in the Del
 
 On a high level:
```

Comment:
"At a high level:"

Reply:
Solved in [3584051](https://github.com/Qbeast-io/qbeast-spark/pull/310/commits/358405183d8d9aa6a357277cc80526fbc92448b1)